src/main/kotlin/no/nb/mlt/wls/infrastructure/repositories/item/MongoItemRepository.kt

```diff
@@ -87,6 +87,24 @@ class ItemRepositoryMongoAdapter(
 
         return getItem(hostName, hostId)!!
     }
+
+    override suspend fun pickItem(item: Item) {
+        mongoRepo
+            .findAndUpdateItemByHostNameAndHostId(item.hostName, item.hostId, 0.0, "Picked")
```

Comment:
Shouldn't we either calculate the quantity here, or get item object with correct quantity?
Also Picked is not a location, we either keep the location as is (if quantity > 0) or WITH_LENDER (if quantity = 0)
The same rules for quantity and location apply to our DB as to what we return in callback to hosts.

Reply:
This function is actually unused at the moment. The functionality for picking was delegated to use `moveItem` calls instead